rinja_derive/src/config.rs

```diff
@@ -360,6 +360,26 @@ impl<'a> RawSyntax<'a> {
             }
         }
 
+        for end in [syntax.block_start, syntax.expr_start, syntax.comment_start] {
+            for prefix in ["<<", ">>", "&&", "..", "||"] {
+                if end.starts_with(prefix) {
+                    let msg = if end == prefix {
+                        format!("a closing delimiter must not start with an operator: {end:?}")
+                    } else {
+                        format!(
+                            "a closing delimiter must not start an with operator: \
+                             {end:?} vs {prefix:?}",
```

Comment:
The "vs" part is a bit weird. I'd reword it like: `a closing/opening delimiter must not start with an operator: {end:?} starts with {prefix:?}`.